components/scream/src/dynamics/homme/interface/homme_params_mod.F90

```diff
@@ -118,6 +118,7 @@ subroutine init_params_f90 (nl_fname_c) bind(c)
     namelist /ctl_nl/test_case, &
       u_perturb,                &
       partmethod,               &         ! mesh partitioning method
+      cubed_sphere_map,         &
```

Comment:
Have you tried running a CIME case? The entry `cubed_sphere_map` is not in `data/scream_input.yaml`, which is used to generate the input namelist. I have still to understand namelist completely, but it's my understanding that if a namelist entry is not found in the input file, then random errors happen. Maybe I'm wrong, but if you could run a case through valgrind, and  verify that there are no errors, and the correct params are printed, that'd be great.

Reply:
I added the entry into scream_input.yaml as a part of this PR and it works through CIME.

The question would be if it is ok for the standalone test (which don't include it in their .yaml files). I'll run those.

@singhbalwinder says that it is necessary for us to set this to 0, maybe he can elaborate here. I copied this from 
`scream/components/homme/src/share/namelist_mod.F90`